ci/docker/ml.build.Dockerfile

```diff
@@ -27,6 +28,11 @@ DOC_TESTING=1 TRAIN_TESTING=1 TUNE_TESTING=1 DATA_PROCESSING_TESTING=1 INSTALL_H
   ./ci/env/install-dependencies.sh
 INSTALL_HDFS=1 ./ci/env/install-dependencies.sh
 
+# setup credentials for postmerge pipeline
+if [[ "$BUILDKITE_PIPELINE_ID" == "0189e759-8c96-4302-b6b5-b4274406bf89" ]]; then
+  python ci/env/setup_credentials.py >> ~/.bashrc
```

Comment:
this is not right.. please do not save secrets inside the container image.

discussed offline, let's get a way to pipe-in these env vars into the container that runs the test

or we can fetch the secrets inside the test (with a fallback to reading it from env var) and remove this script.